packages/indexer/indexer/db.py

```diff
@@ -0,0 +1,168 @@
+import json
+import aiohttp
+
+import asyncpg
+from indexer.chain_mapper import CosmosChain
+from indexer.data import get_block
+
+
+async def drop_tables(pool: asyncpg.pool):
+    await pool.execute(
+        """
+        DROP TABLE IF EXISTS raw CASCADE;
+        DROP TABLE IF EXISTS blocks CASCADE;
+        DROP TABLE IF EXISTS txs CASCADE;
+        """
+    )
+
+
+async def create_tables(pool: asyncpg.pool):
```

Comment:
These queries I think should exist in `.sql` files and are then read-in. In-line makes it harder to read what the code is doing.